company-policies/annual-review-process.md

```diff
@@ -1,7 +1,3 @@
----
-title: Annual performance review
----
```

Comment:
@elizabethraley we need to keep this section for the menus to work.

Reply:
Added this back